compiler/q-implant-op-level-test/q-implant-op-level-test.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
I'm not sure about this folder name `import`.
what is your intention of this folder?

Reply:
This folder name means to import test data.
So I named the folder `import`.

Reply:
Please add `import/README.md` file about the purpose of this folder, how to add new new test items.

Reply:
> This folder name means to import test data.

this `q-implant-op-level-test` module is to test `q-implant` by operator level.
as I understand this `import` folder is to generate data files to test `q-implant` tool.
from this perspective, I'm confused of the word `import` testing of `q-implant`. 
what do you mean by `import` here?

Reply:
I intented to name this folder `import` to put class files to load while testing.
I think `import` can be renamed to `recipes` or `examples`.

Reply:
> can be renamed to recipes or examples.

there is existing term `recipe` for tflchef or circlechef tool, that will also make confusion.
I don't think `examples` is appropriate as it will express this folder is an example of some operator tests, which I don't think is your intention.

Reply:
I don't have no idea.
Can you recommand appropriate name?

Reply:
Don't give up! take a coffee, look outside ...

Reply:
I brought some ideas
1. blueprint : These files can be said to be blueprints for taking q-params.
2. operator : These files contain information from each operator.
3. mold : These files have only a mold, and q-params are output through the operation of the script.

Reply:
while looking to the module name, `op-level-test`, most of our integration tests are done in operator levels. so, just `q-implant-test` or `q-implant-blabla-test` seems more appropriate.

then for the word `blabla` ...
what test do we do here? as I understand, the main test here is to implant quantization parameters and check if they are implanted correctly.

so, for the module name `q-implant-qparam-test` seems to express WHAT this module is doing.
and then, for the folder name, the files (__init__.py) are providing these quantization parameters per operators (and attribute variations).

from this,
- `blueprint` and `mold`: a bit ambiguous to me
- `operator`: expresses that this folder contains test data per operators
- `q-params`: you used `q-params` twice in `blueprint` and `mold` which I think directly expresses what the folder holds

if there are any misunderstanding, please point out.

Reply:
from https://github.com/Samsung/ONE/pull/11579#discussion_r1346607634

`testcase` also looks OK.

@jinevening , any comments?

Reply:
`q-implant-qparam-test` and `q-params` look good to me.

@FantBlog Could you change the module name into `q-implant-qparam-test` and the directory name into `q-params`? And please post PRs ASAP. We don't have much time left :)

Reply:
@jinevening Rename is done. :)